docs/management/admin/endpoint-protection-rules.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[endpoint-protection-rules]]
+= Endpoint protection rules
+
+Endpoint protection rules are prebuilt rules designed to help you manage and respond to alerts generated by {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention. These rules include the <<endpoint-security, {elastic-defend}>> rule as well as additional detection and prevention rules for different {elastic-defend} protection features.
+
+NOTE: To receive {elastic-endpoint} alerts, you must install the Endpoint agent on your hosts (see <<install-endpoint>>).
+
+[discrete]
+== {elastic-defend} rule
```

Comment:
While anchors aren't necessary, they are nice to have in case you ever need to link to this section from another page.